crates/cli/src/tasks/csharp.rs

```diff
@@ -11,6 +12,14 @@ pub(crate) fn build_csharp(project_path: &Path, _build_debug: bool) -> anyhow::R
         std::fs::remove_file(&output_path)?;
     }
 
+    // Ensure the project path exists
+    if fs::metadata(project_path).is_err() {

```

Comment:
Why are we not returning the actual error? Are we sure there's only one error that can be returned?